### ISSUE TITLE
Not supported in Monterey?

### ISSUE THREAD

**Issue by crstauf** [chrstphrknwtn/epoch-flip-clock-screensaver#8]
<img width="273" alt="Screen Shot 2021-11-02 at 22 08 31" src="https://user-images.githubusercontent.com/4573033/140000911-ffc7ddd0-592a-4b44-93c2-1cac93f5a0f5.png">

**Comment by lury** [comment]
You have to go to security settings after trying to launch this screen saver and allow opening it
![CleanShot 2022-02-03 at 23 36 10](https://user-images.githubusercontent.com/457190/152440990-d5a042f7-0c38-43e3-87cb-82d90a49d8f3.jpg)

**Comment by crstauf** [comment]
@lury Yes, I'm aware I can bypass the warning, but why not allow Apple to check?
